doc/architecture/decisions/0002-migration-and-sync-with-NOMIS.md

```diff
@@ -98,17 +98,158 @@ This endpoint will return all the adjudication information needed to populate NO
+      "victimStaffUsername": "ABC12D",
+      "victimOtherPersonsName": "Bob Hope"
+    },
+    "incidentStatement": {
+      "statement": "string",
+      "completed": true
+    },
+    "createdByUserId": "string",
+    "createdDateTime": "2023-03-28T12:29:35.791Z",
+    "status": "ACCEPTED",
+    "reviewedByUserId": "string",
+    "statusReason": "string",
+    "statusDetails": "string",
+    "damages": [
+      {
+        "code": "CLEANING",
+        "details": "the kettle was broken",
+        "reporter": "ABC12D"
+      }
+    ],
+    "evidence": [
+      {
+        "code": "PHOTO",
+        "identifier": "Tag number or Camera number",
+        "details": "ie what does the photo describe",
+        "reporter": "ABC12D"
+      }
+    ],
+    "witnesses": [
+      {
+        "code": "PRISON_OFFICER",
+        "firstName": "Fred",
+        "lastName": "Kruger",
+        "reporter": "ABC12D"
+      }
+    ],
+    "hearings": [
+      {
+        "id": 0,
+        "locationId": 0,
+        "dateTimeOfHearing": "2023-03-28T12:29:35.791Z",
+        "oicHearingType": "GOV_ADULT",
+        "outcome": {
+          "id": 0,
+          "adjudicator": "string",
+          "code": "COMPLETE",
+          "reason": "LEGAL_ADVICE",
+          "details": "string",
+          "plea": "UNFIT"
+        }
+      }
+    ],
+    "issuingOfficer": "string",
+    "dateTimeOfIssue": "2023-03-28T12:29:35.791Z",
+    "disIssueHistory": [
+      {
+        "issuingOfficer": "string",
+        "dateTimeOfIssue": "2023-03-28T12:29:35.791Z"
+      }
+    ],
+    "dateTimeOfFirstHearing": "2023-03-28T12:29:35.791Z",
+    "outcomes": [
+      {
+        "hearing": {
+          "id": 0,
+          "locationId": 0,
+          "dateTimeOfHearing": "2023-03-28T12:29:35.791Z",
+          "oicHearingType": "GOV_ADULT",
+          "outcome": {
+            "id": 0,
+            "adjudicator": "string",
+            "code": "COMPLETE",
+            "reason": "LEGAL_ADVICE",
+            "details": "string",
+            "plea": "UNFIT"
+          }
+        },
+        "outcome": {
+          "outcome": {
+            "id": 0,
+            "code": "REFER_POLICE",
+            "details": "string",
+            "reason": "ANOTHER_WAY",
+            "amount": 0,
+            "caution": true,
+            "quashedReason": "FLAWED_CASE"
+          },
+          "referralOutcome": {
+            "id": 0,
+            "code": "REFER_POLICE",
+            "details": "string",
+            "reason": "ANOTHER_WAY",
+            "amount": 0,
+            "caution": true,
+            "quashedReason": "FLAWED_CASE"
+          }
+        }
+      }
+    ],
+    "punishments": [
+      {
+        "type": "PRIVILEGE",
+        "privilegeType": "CANTEEN",
+        "otherPrivilege": "string",
+        "stoppagePercentage": 0,
+        "schedule": {
+          "days": 0,
+          "startDate": "2023-03-28",
+          "endDate": "2023-03-28",
+          "suspendedUntil": "2023-03-28"
```

Comment:
a punishment schedule either has a suspendedUntil date, or a start and end date.  it wont have all three set